staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -932,9 +937,16 @@ var (
 		nodeIncludedPredicate,
 		nodeUnTaintedPredicate,
 	}
+	stableNodeSetPredicates []NodeConditionPredicate = []NodeConditionPredicate{
```

Comment:
Yeah, I did this because I wanted the code to read better should something go wrong with this feature and some developer starts going through the code to debug. Keeping `etpLocalNodePredicates` and `stableNodeSetPredicates` separate, was my intention of indicating what has changed w.r.t this feature and what was there before (i.e: that these were already the applied predicates for eTP:Local).    

This might not be a valid use case?

Reply:
I'm OK either way.  I see your point.